### ISSUE TITLE
Double command execution

### ISSUE THREAD

**Issue by bretonium** [necropotame/teeworlds-infclass#217]
1. Login as moderator/aministrator
2. Execute command "status"

Expected: the command is executed once
Actual: the command is executed twice

In commit 09e02021 the following lines were added:
`
diff --git a/src/engine/shared/console.cpp b/src/engine/shared/console.cpp
index c8711f9..2d7398b 100644
--- a/src/engine/shared/console.cpp
+++ b/src/engine/shared/console.cpp
@@ -372,6 +372,10 @@ void CConsole::ExecuteLineStroked(int Stroke, const char *pStr, int ClientID, bo
                                                        Print(OUTPUT_LEVEL_STANDARD, "Console", "Invalid arguments.");
                                                        Print(OUTPUT_LEVEL_STANDARD, "Console", aBuf);
                                                }
+                                               else
+                                               pCommand->m_pfnCallback(&Result, pCommand->m_pUserData);
+
+                                               return;
                                        }
                                }
                        }
`

This leads to double execution of commands in console. Any idea why it was added? Can we remove it?

**Comment by bretonium** [comment]
Oh well.
```
diff --git a/src/engine/shared/console.cpp b/src/engine/shared/console.cpp
index c8711f9..2d7398b 100644
--- a/src/engine/shared/console.cpp
+++ b/src/engine/shared/console.cpp
@@ -372,6 +372,10 @@ void CConsole::ExecuteLineStroked(int Stroke, const char *pStr, int ClientID, bo
                                                        Print(OUTPUT_LEVEL_STANDARD, "Console", "Invalid arguments.");
                                                        Print(OUTPUT_LEVEL_STANDARD, "Console", aBuf);
                                                }
+                                               else
+                                               pCommand->m_pfnCallback(&Result, pCommand->m_pUserData);
+
+                                               return;
                                        }
                                }
                        }

```

**Comment by progval** [comment]
So that's what was causing that bug! Thanks! Fixed in 1e9d7be9a.
